go.mod

```diff
@@ -109,9 +109,9 @@ require (
 	go.opentelemetry.io/otel/sdk/log v0.8.0 // indirect
 	go.opentelemetry.io/otel/sdk/metric v1.32.0 // indirect
 	go.opentelemetry.io/proto/otlp v1.3.1 // indirect
-	golang.org/x/sync v0.9.0
-	golang.org/x/sys v0.27.0 // indirect
-	golang.org/x/text v0.20.0 // indirect
+	golang.org/x/sync v0.10.0
```

Comment:
Looks like this one ended up in the wrong group at some point (it's a direct dependency, but now stuffed in the list of indirects).

Reply:
Yeah we need to update it in some followup.